tests/translator/test_translator.py

```diff
@@ -265,57 +267,57 @@ class TestTranslatorEndToEnd(TestCase):
       ] # Run all the above tests against each of the list of partitions to test against
       )
     )
-    @patch('samtranslator.plugins.application.serverless_app_plugin.ServerlessAppPlugin._sar_service_call', mock_sar_service_call)
-    @patch('botocore.client.ClientEndpointBridge._check_default_region', mock_get_region)
     def test_transform_success(self, testcase, partition_with_region):
-        partition = partition_with_region[0]
-        region = partition_with_region[1]
-
-        manifest = yaml_parse(open(os.path.join(INPUT_FOLDER, testcase + '.yaml'), 'r'))
-        # To uncover unicode-related bugs, convert dict to JSON string and parse JSON back to dict
-        manifest = json.loads(json.dumps(manifest))
-        partition_folder = partition if partition != "aws" else ""
-        expected_filepath = os.path.join(OUTPUT_FOLDER, partition_folder, testcase + '.json')
-        expected = json.load(open(expected_filepath, 'r'))
-
-        with patch('boto3.session.Session.region_name', region):
-            parameter_values = get_template_parameter_values()
-            mock_policy_loader = MagicMock()
-            mock_policy_loader.load.return_value = {
-                'AWSLambdaBasicExecutionRole': 'arn:{}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole'.format(partition),
-                'AmazonDynamoDBFullAccess': 'arn:{}:iam::aws:policy/AmazonDynamoDBFullAccess'.format(partition),
-                'AmazonDynamoDBReadOnlyAccess': 'arn:{}:iam::aws:policy/AmazonDynamoDBReadOnlyAccess'.format(partition),
-                'AWSLambdaRole': 'arn:{}:iam::aws:policy/service-role/AWSLambdaRole'.format(partition),
-            }
+        with (patch('samtranslator.plugins.application.serverless_app_plugin.ServerlessAppPlugin._sar_service_call', mock_sar_service_call),
```

Comment:
Thank you for this! I went ahead and fixed the merge conflicts that were happening in this file; could you make sure I did it right?

This should also fix the weird "unbound local variable" error that kept happening; I worked out a similar fix recently but you beat me to it 👍